src/main/webapp/auth.js

```diff
@@ -116,7 +117,7 @@ function signOut() {
 function renderButton() {
   gapi.signin2.render('google-sign-in-btn', {
     'scope': 'https://www.googleapis.com/auth/gmail.readonly ' +
-        'https://www.googleapis.com/auth/calendar.readonly ' +
+        'https://www.googleapis.com/auth/calendar ' +
```

Comment:
I don't see any post methods built out yet for calendar. Is there any particular reason why this needs to be changed?

Reply:
For later use. We did specify in the design doc that we wanted to give `read` and `write` authorization to this API. I don't think there is necessarily a benefit in doing it later than now.

Reply:
True - It doesn't really matter too much. We can leave it